docs/getting-started-guides/aws/cloudformation-template.json

```diff
@@ -113,121 +113,177 @@
         "SecurityGroups": [{"Ref": "KubernetesSecurityGroup"}],
         "UserData": { "Fn::Base64": {"Fn::Join" : ["", [
           "#cloud-config\n\n",
+          "---\n",
+          "write_files:\n",
+          "- path: /opt/bin/waiter.sh\n",
+          "  owner: root\n",
+          "  content: |\n",
+          "    #! /usr/bin/bash\n",
+          "    until curl http://127.0.0.1:4001/v2/machines; do sleep 2; done\n",
           "coreos:\n",
```

Comment:
the next few lines don't appear in your modified node.yaml or master.yaml in this PR.  Is that on purpose?